demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -120,6 +120,25 @@ def is_docker_image_same_as_yml(self) -> bool:
 
         return True
 
```

Comment:
Please add the error_codes decorator with the corresponding error code of the validation:
`@error_codes('ERROR_CODE')`